### PR TITLE
Switch to TravisCI

### PR DESCRIPTION
**Pull request by Baggz**
I would like to test multiple Node.js versions and that's unfortunately not possible with CircleCI.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+language: node_js
+
+node_js:
+  - "4.1"
+  - "4.0"
+  - "0.12"
+  - "0.11"
+  - "0.10"
```
Comment (XVincentX) [inline .travis.yml]: I would really not test against 0.10 as it does not support a lot of really important ES6 features.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+language: node_js
+
+node_js:
+  - "4.1"
+  - "4.0"
+  - "0.12"
+  - "0.11"
```
Comment (XVincentX) [inline .travis.yml]: I would totally skip 0.11 as it is an unstable version by definition.

**Comment by XVincentX** [comment]
CircleCI should support multiple builds as well. I'll get back to you ASAP.

**Comment by Baggz** [comment]
@XVincentX 

> CircleCI should support multiple builds as well. I'll get back to you ASAP.

Seems to be possible via the following.

```
test:
  override:
    - nvm use 0.10 && ./runtest
```

Build artifacts might be a good reason why stay with CircleCI.

File: .travis.yml (hunk 2 above)
Comment (Baggz) [inline .travis.yml]: @XVincentX 

> I would totally skip 0.11 as it is an unstable version by definition.

Okay, agree, will fix.

File: .travis.yml (hunk 1 above)
Comment (Baggz) [inline .travis.yml]: @XVincentX 

We are not there yet, Apiary still runs 0.10; as long as Apiary runs 0.10 we have to fully support the version.

**Comment by Baggz** [comment]
@XVincentX 

Let's stick with Circle, mainly because of the build artifacts support.

**Pull request closed without merging**